Tests/Library/Client.tests.ts

```diff
@@ -306,7 +315,7 @@ describe("Library/Client", () => {
                 assert.ok(trackStub.calledOnce);
                 var args = trackStub.args;
                 assert.equal(args[0][0].baseType, "Microsoft.ApplicationInsights.RequestData");
-                assert.equal(args[0][0].baseData.responseCode, 200);
```

Comment:
It's not gone, but it is not 200 anymore. This test is simulating a failed request, so the `responseCode` never should have been 200. My refactor of the `RequestDataHelper`/`RequestParser` class fixed this bug.